affirm/src/main/java/com/affirm/android/Affirm.java

```diff
@@ -44,9 +44,11 @@ private Affirm() {
     public static final int LOG_LEVEL_ERROR = Log.ERROR;
     public static final int LOG_LEVEL_NONE = Integer.MAX_VALUE;
 
-    private static final int CHECKOUT_REQUEST = 8076;
-    private static final int VCN_CHECKOUT_REQUEST = 8077;
-    private static final int PREQUAL_REQUEST = 8078;
+    // these values are set by the builder
+    private static int CHECKOUT_REQUEST;
+    private static int VCN_CHECKOUT_REQUEST;
+    private static int PREQUAL_REQUEST;
+
     static final int RESULT_ERROR = -8575;
```

Comment:
Can the RESULT_ERROR code be configurable also?

Reply:
@AmyBeall this field will not cause any conflict with the host application.  There shouldn't be any benefit to make this configurable.  Is it okay to leave this as is?

Reply:
Ok great, that's fine then. I will go ahead and approve this.